web/src/Pages/Text/Search.elm

```diff
@@ -157,6 +162,9 @@ type Msg
     | CloseHint TextSearch.Help.TextHelp
     | PreviousHint
     | NextHint
+      -- rating messages
+    | CastVote Vote Int
+    | Vote (Result (Http.Detailed.Error String) ( Http.Metadata, VoteResponse ))
```

Comment:
Let's rename this `GotVote` and rename `CastVote` to `Vote`.

Reply:
changed in commit 5612469